cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/validator/AwsLogRolePermissionValidator.java

```diff
@@ -107,6 +107,13 @@ public void validate(AmazonIdentityManagementClient iam, InstanceProfile instanc
                     String.join(", ", instanceProfile.getArn()),
                     String.join("\n", failedActions));
 
+            if (validationErrorMessage.contains(AbstractAwsSimulatePolicyValidator.DENIED_BY_ORGANIZATION_RULE)) {
+                validationErrorMessage = String.join(validationErrorMessage,
```

Comment:
I assume you wanted to use `String.format` or concat.
Join will not work in this case. The original message will be lost.

Reply:
Also no test case for this. (no test class)

Reply:
Yes, good catch, thank you. I will create the test as well.